concrete/src/Workflow/BasicWorkflow.php

```diff
@@ -141,11 +142,11 @@ public function start(WorkflowProgress $wp)
                 $ui = UserInfo::getByID($req->getRequesterUserID());
 
                 // let's get all the people who are set to be notified on entry
-                $message = t(
+                $message = array(
```

Comment:
The translation system (that populates the strings to be translated in [translate.concrete5.org](https://translate.concrete5.org]) looks for `t()` calls to see which strings must be translated.
If you remove the `t` call around a string, that string will be excluded from the translatable string list.